plots/generate-dashboard.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+const opn = require('opn');
+const args = require('yargs')
+  .example('node $0 --input-root dir-with-subdirectories')
+  .example('node $0 --inputs out-1 out-2 out-3')
+  .describe({
+    'inputs': 'paths to out directory generated by measure.js',
```

Comment:
could we instead have a single `--input`? (And tbh, at that point, you might as well skip the arg).

It'd look like this:

```sh
node generate-dashboard out-a out-b out-x

node generate-dashboard outParentFolder
```

Take the directory or its children, depending on the contents